scripts/migration/o19/generate_manifests.py

```
@@ -0,0 +1,914 @@
+
+
+# keys whose stock value is a credential. Their defaults are NEVER emitted
+# into the shipped manifest (a plaintext password in a package is a
+# finding, whatever its origin); the props phase always surfaces such a
+# key for review instead of baseline-diffing it.
+SECRET_KEY_RE = re.compile(
+    r"(^|[._])(password|passwd|pass|secret|api_key|apikey|conformancekey|"
+    r"token|pgp_key|user|username|userid)$", re.I)
+
+
+def is_secret_key(key: str, ov_props=None) -> bool:
```

Ruff (0.16.3)

[warning] Unused function argument: `ov_props`

(ARG001)